src/modules/fundingManager/oracle/interfaces/IOraclePrice_v1.sol

```diff
@@ -35,10 +35,10 @@ interface IOraclePrice_v1 {
     /// @notice Gets current price for token issuance (buying tokens)
     /// @return price_ Current price in 18 decimals (collateral tokens per 1 issuance token)
```

Comment:
This comment might not be accurate, as we actually denormalize the decimals before returning them. Same for the `getPriceForRedemption()` function

Reply:
corrected